src/pages/books/Books.jsx

```diff
@@ -4,32 +4,38 @@ import BookCard from '../../components/BookCard';
 import Style from './Books.module.css';
 import BookForm from '../../Forms/book-form/BookForm';
 import { removeBook, addBook, fetchBooks } from '../../redux/books/booksSlice';
+import Loader from '../../components/Loader';
 
 /* eslint-disable react-hooks/exhaustive-deps */
 export default function Books() {
-  const { books } = useSelector((state) => state.book);
+  const { books, work } = useSelector((state) => state.book);
   const dispatch = useDispatch();
   useEffect(() => {
     dispatch(fetchBooks());
   }, []);
 
   return (
-    <>
+    <div className={`${Style.BookPage}`}>
       <div className={`${Style.PageContainer}`}>
         {
-          books.map((book) => (
```

Comment:
Good job working on the project requirements but i noticed you disaled eslint it is against microverse policy to disable linters so it would be nice if you enabled eslint in this file.    Please do this in all files were eslint was disabled.